src/pages/issues/index.tsx

```diff
@@ -86,12 +83,9 @@ const Issues: FC = () => {
     };
 
     useEffect(() => {
-        fetchIssues();
+        fetchIssues('');
     }, []);
```

Comment:
why are we passing empty string over here

Reply:
any specific reason?

Reply:
don't you think we could have passed default argument as empty string instead of passing empty string as a parameter.

Reply:
Yeah. It's a small change I will accommodate in some other PR. 
Thank you for your time

Reply:
Considering the fact that this change doesn't seem to enhance functionality or readability, I don't believe it is necessary to include it. Moreover, going through the approval process again for this change would considerably delay the agreed-upon timelines.

However, I want to assure you that I will address this issue in my next pull request on the status repository.

I hope this rationale aligns with your understanding and that you agree with my thought process.

Reply:
Also, I guess you would need to take re-approvals because today some changes were merged in the develop branch in the Layout file so request you to please take a fresh pull and then resolve the merge conflicts and make necessary changes and the changes that I have suggested